pkg/extensions/enterprise_imports.go

```diff
@@ -32,14 +32,13 @@ import (
 	_ "golang.org/x/time/rate"
 	_ "xorm.io/builder"
 
-	_ "github.com/grafana/authlib/authn"
-	_ "github.com/grafana/authlib/authz"
-	_ "github.com/grafana/authlib/cache"
-	_ "github.com/grafana/authlib/grpcutils"
-	_ "github.com/grafana/authlib/types"
```

Comment:
Is this intentional?

Reply:
not at all, will revert this removal

Reply:
It still keeps doing that in my local dev - whenever enterprise is linked. I am not sure why.

Reply:
its possible, it is actually legit and its removing them because devs have started using these subpackages of authlib in OSS itself. iiuc, enterprise_imports only needs to include packages that are only used by enterprise and not OSS? Would you concur?